doc/pages/user/015-astarte_dashboard.md

```diff
@@ -0,0 +1,394 @@
+blocks, both custom and native ones: the former ones are those defined by you, the latter ones are
+those provided by Astarte and are displayed with a `native` label.
+
+### Creating blocks
+
+From the Block list, clicking on the **Create** button will load a dedicated page to define a new
+block.
+
+Here you can define the block by specifying:
+
+- A `name` for the block.
+- A `type`, between `Producer`, `Consumer` or `Producer & Consumer`, depending on how the block
+  should treat messages and connect to other blocks.
+- A `schema`, reporting the JSON Schema definition of how a configuration should look like for the
+  block.
+- A `source`, containing the block's implementation, written in the
+  [DSL format](https://docs.astarte-platform.org/flow/snapshot/0010-defining-a-pipeline.html) for
+  Pipelines.
+
+Confirm the block's creation by clicking on the **Create new block** button.
+
+To learn more about blocks definition you can read the
+[documentation's reference](https://docs.astarte-platform.org/flow/snapshot/0003-blocks.html).
+
+### Editing blocks
+
+From the Block list, you can select a block to view its definition in a dedicated page.
+
+Here you can review the block's details such as the block's type, the schema and, if it is a custom
+block, its source.
+
+To delete a block, click on the **Delete block** button. Note that you cannot delete `native` blocks
+provided by Astarte.
+
+## Pipelines
+
+A [Pipeline](https://docs.astarte-platform.org/flow/snapshot/0004-pipelines.html) is a computation
+blueprint (therefore a description) built as a chain of blocks.
+
+You can navigate to this section thanks to the Dashboard's side menu. Here is a list of all existing
+pipelines from which you can review, manage and
+[instantiate](015-astarte_dashboard.html#instantiating-flows) them.
+
+### Creating pipelines
+
+From the Pipeline list, clicking on the **Create** button will load up the **Pipeline Editor** page,
+an interactive tool that you can use to configure your pipeline.
+
+The Pipeline Editor is composed of two parts:
+
+- A sidebar listing all available Blocks, grouped by type
+- A space where you can drag & drop blocks, connecting them to effectively design a pipeline
+
+You can read more about the Pipeline Editor on the
+[dedicated documentation](https://docs.astarte-platform.org/flow/snapshot/0020-pipeline-editor.html).
+
+Once you have designed your pipeline, you can review and specify its:
+
+- `name`
+- `source`, containing the pipeline's implementation, written in the
+  [DSL format](https://docs.astarte-platform.org/flow/snapshot/0010-defining-a-pipeline.html) for
+  Pipelines.
+- `schema`, reporting the JSON Schema definition of how a configuration should look like for the
+  pipeline.
+- `description`, explaining the scope and supposed usage of the pipeline.
+
+Then hit the **Create new pipeline** button to confirm the pipeline's definition.
+
+### Editing pipelines
```

Comment:
Likewise for pipelines, they can't be edited

Reply:
Fixed it